src/Type/PhpDecimalOperatorTypeSpecifyingExtension.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace PHPStan\Type;
+
+final class PhpDecimalOperatorTypeSpecifyingExtension implements OperatorTypeSpecifyingExtension
+{
+
+	public function isOperatorSupported(string $operatorSigil, Type $leftSide, Type $rightSide): bool
+	{
+		return in_array($operatorSigil, ['-', '+', '*', '/'])
```

Comment:
3rd parameter should be true. Also, I'd rather make this only part of `tests/` and having it be tested somehow.